src/Compilers/CSharp/Portable/CodeGen/EmitConversion.cs

```diff
@@ -111,6 +111,10 @@ private void EmitConversion(BoundConversion conversion)
 
                     _builder.EmitNumericConversion(fromPredefTypeKind, toPredefTypeKind, conversion.Checked);
                     break;
+                case ConversionKind.PinnedObjectToPointer:
+                    // same representation, but stop GC tracking
```

Comment:
Will update the comment.

What I meant was something like:

CLR allows unsafe conversion from (O) to native int/uint. 
The conversion does not change the representation of the value, but the value will not be reported to subsequent garbage-collection operations (and therefore will not be updated by such operations)

---
In reply to: [125134566](https://github.com/dotnet/roslyn/pull/20548#discussion_r125134566) [](ancestors = 125134566)